modules/statistics/LocalStatsCollector.js

```diff
@@ -142,5 +143,45 @@ LocalStatsCollector.prototype.stop = function() {
  * @returns {boolean}
  */
 LocalStatsCollector.isLocalStatsSupported = function() {
-    return Boolean(context);
+    return Boolean(window.AudioContext);
 };
+
+/**
+ * Disconnects the audio context.
+ */
+LocalStatsCollector.disconnectAudioContext = async function() {
+    if (context) {
+        logger.info('Disconnecting audio context');
+        await context.close();
+        context = null;
+    }
+};
+
+/**
+ * Connects the audio context.
+ */
+LocalStatsCollector.connectAudioContext = function() {
+    if (!LocalStatsCollector.isLocalStatsSupported()) {
+        return;
+    }
+
+    logger.info('Connecting audio context');
+    context = new AudioContext();
+
+    // XXX Not all browsers define a suspend method on AudioContext. As the
+    // invocation is at the (ES6 module) global execution level, it breaks the
+    // loading of the lib-jitsi-meet library in such browsers and, consequently,
+    // the loading of the very Web app that uses the lib-jitsi-meet library. For
+    // example, Google Chrome 40 on Android does not define the method but we
+    // still want to be able to load the lib-jitsi-meet library there and
+    // display a page which notifies the user that the Web app is not supported
+    // there.
+    if (context.suspend) {
+        context.suspend();
+    }
+};
+
+/**
+ * Initialize the audio context on startup.
+ */
+LocalStatsCollector.connectAudioContext();
```

Comment:
Since this would now return a promise (see above) we should store it in some `initContext` variable start awaits on, to make sure it's initialized when we start.

Reply:
Discussed that the promise won't resolve until resume is called, so we're dumping it.